cmd/config-downloader/downloader.go

```diff
@@ -150,7 +150,7 @@ func main() {
 	region = util.DetectRegion(mode, cc.CredentialsMap())
 
 	if region == "" && downloadLocation != locationDefault {
-		log.Println("E! Unable to determine aws-region.")
+		fmt.Println("Unable to determine aws-region.")
```

Comment:
**Non blocker**: Keep the original message is better for me since customers are familiar with search for error and E! is a flag for that